### ISSUE TITLE
harvest.Harvest.clients() is_active bool returns inactive clients regardless of value

### ISSUE THREAD

**Issue by gshel** [bradbase/python-harvest_apiv2#15]
Hope you're doing well! I came across this bug when testing whether or not I can return a list of only active clients using [Harvest.clients()](https://github.com/bradbase/python-harvest_apiv2/blob/05b02b90d4247557d57abfdbb259b7e70c91a90d/harvest/harvest.py#L106). As a user, I would expect the class's method to work like this:
- `Harvest.clients(is_active=True)` returns active clients
- `Harvest.clients(is_active=False)` returns inactive clients

What I actually get, regardless of the boolean passed, is both the active and inactive clients. I tried regular boolean values and string versions of a boolean but received the same response both times.

![harvest-python-v2 is_active](https://user-images.githubusercontent.com/35184207/97738901-6290ed80-1ab5-11eb-8587-59cc719f8869.jpg)

Could you let me know if I'm using this in the wrong way, or if it's a bug? If it's a bug, I'll be happy to fix it.

**Comment by bradbase** [comment]
Hi @gshel

Things are going well thank you for asking :D

I hope the reporting project is going well.

Great pick-up BTW. This is a fun one.

You are using the library as I expect people to.

I have used boolean datatype for the is_active parameter. The request you are triggering with python-harvest_apiv2 generates a URL like this;
```
https://api.harvestapp.com/api/v2/clients?page=1&per_page=100&is_active=False
```
It appears as though Harvest have built their API to be case sensitive and not tolerant of different cases. I have found this works;
```
https://api.harvestapp.com/api/v2/clients?page=1&per_page=100&is_active=false
```
So... Who's bug is it? I have an opinion on that but I can't impact such a change. To get you a URL that works python-harvest_apiv2 needs to cast booleans currently used in URL parameters to strings and transform them to lower case while constructing the URL. This fix will need to be project wide.

**Comment by bradbase** [comment]
@gshel Are you still interested in fixing the bug?

**Comment by gshel** [comment]
I am, yes! I don't have a ton of time during the week, so I'll either take a look at this on Friday or I have a colleague that's learning Python who may take a stab at this. If my colleague grabs it, I'll work with them to get a nice PR put together for you.

**Comment by bradbase** [comment]
@gshel Huzzah! Thank you

Nobody else has been vocal about the bug so the timing is yours.

I 100% support delegating to your learner if they consent and you've got the time (and, I guess if they're on the clock, the money). I like to think this project is great for beginners and, more so, this bug is brilliant for learning to code (and learning how the web works 😄 ).

**Comment by gshel** [comment]
Hey opening a branch for this now!

**Comment by bradbase** [comment]
Thanks @gshel 

Your contribution is well valued.

Has been released in 1.11.0
